lib/game.rb

```diff
@@ -0,0 +1,18 @@
+class Game
+  def initialize(input:, output:)
+    @input = input
+    @output = output
+  end
+
+  def play
```

Comment:
One thing I have been thinking about is in my own fork is that this starts out breaking the design rules. This is a method with > 5 files. I think changes now influence the direction of the architecture in the future, so it is a really interesting moment to think about a refactor. 

Might be worthwhile thinking about how you would change this to meet design rules. Even if it seems like a bad idea for this PR, it is an interesting thought exercise.

Reply:
That's a good point! I didn't even consider the 5 line rule here - that may be because I saw it was a single puts statement and shrugged it off. I do like the idea of sticking with the rules from the start. I'll look into adjusting this.

Reply:
Moved the grid to a constant in f06c167 - loose thinking is to use that as an `sprintf` style template, but holding off on figuring that out for now.